client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -257,6 +259,27 @@ export const ApplicationsTable: React.FC = () => {
     onDeleteApplicationSuccess,
     onDeleteApplicationError
   );
+  const onHandleCopySuccess = (hasSuccessfulReviewCopy: boolean) => {
+    setIsSubmittingBulkCopy(false);
+    pushNotification({
+      title: hasSuccessfulReviewCopy
+        ? t("toastr.success.assessmentAndReviewCopied")
+        : t("toastr.success.assessmentCopied"),
+      variant: "success",
+    });
+  };
+  const onHandleCopyError = (error: AxiosError) => {
+    setIsSubmittingBulkCopy(false);
+    pushNotification({
+      title: getAxiosErrorMessage(error),
+      variant: "danger",
+    });
+  };
+
+  const { mutate: createCopy, isCopying } = useCreateBulkCopyMutation({
```

Comment:
This is the new bulk copy mutation that houses the new query polling logic which replaces the [submit logic here](https://github.com/konveyor/tackle2-ui/pull/485/files#diff-0631a29b250364a7dd66110d8c5567ed72ef22f5091b180a8be69fa818799bbdL295)
as well as the polling logic in useFetch